src/hinter/cwd_aware.rs

```diff
@@ -39,15 +39,29 @@ impl Hinter for CwdAwareHinter {
                         Err(err)
                     }
                 })
-                .expect("todo: error handling")
-                .get(0)
-                .map_or_else(String::new, |entry| {
-                    entry
-                        .command_line
-                        .get(line.len()..)
-                        .unwrap_or_default()
-                        .to_string()
-                })
+                .unwrap_or_else(|_| vec![]);
```

Comment:
Good call! That should eliminate a possible failure path! And yeah having no hint wouldn't be a problem if there is an error internally.

As `Vec` should implement `Default` you could also write

```suggestion
                .unwrap_or_default();
```

Reply:
Thanks, done